spikes/lk_boost_ut/tests/test_parametrized.cpp

```diff
@@ -0,0 +1,50 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <tuple>
+
+#include "./tests/config.h"
+#include "boost/ut.hpp"
+
+using namespace boost::ut;
+
+// ? Example taken from:
+// ? https://github.com/boost-ext/ut/blob/master/example/parameterized.cpp
+
+suite suite_parameterized = [] {
+	for (auto i: std::vector {1, 2, 3}) {
+		test("parametrized: for loop / val = " + std::to_string(i)) = [i] { expect(that % i > 0); };
```

Comment:
What `that` refers to?

Reply:
it refers to "nothing", it's just syntaxic sugar to make tests more readable